server/chat-commands/info.ts

```diff
@@ -2604,6 +2611,44 @@ export const commands: ChatCommands = {
 };
 
 export const pages: PageTable = {
+	battlerules(query, user) {
+		const rules = Object.values(Dex.data.Formats).filter(rule => rule.effectType !== "Format");
+		const tourHelp = `https://www.smogon.com/forums/threads/pok%C3%A9mon-showdown-forum-rules-resources-read-here-first.3570628/#post-6777489`;
+		this.title = "Custom Rules";
+		let rulesHTML = `<div clas="pad"><h1>Custom Rules in challenges and tournaments</h1>`;
+		const basics = [
+			`<p>Pok&eacute;mon Showdown! supports custom rules in three ways:</p>`,
+			`<ul><li>Challenging another user, using the command <code>/challenge USERNAME, FORMAT @@@ RULES</code></li>`,
+			`<li>Tournaments, using the command <code>/tour rules RULES</code> (see the <a href="${tourHelp}">Tournament command help)</a></li></ul>`,
+			`<h2><u>Bans</u></h2>`,
+			`<p>Bans are just a <code>-</code> followed by the thing you want to ban.</p>`,
+			`<h3>Individual Bans</h3>`,
+			`<ul><li><code>- Arceus</code> ban a Pok&eacute;mon (including all formes)</li>`,
+			`<li><code>- Arceus-Flying</code> or <code>- Giratina-Altered</code> ban a Pok&eacute;mon forme</li>`,
+			`<li><code>- Baton Pass</code> ban a move</li></ul>`,
+			`<h3>Group Bans</h3>`,
+			`<ul><li><code>- OU</code> or <code>- DUU</code> Ban a tier</li>`,
+			`<li><code>- Mega</code> or <code>- CAP</code> Ban a Pok&eacute;mon Category</li></ul>`,
+			`<h3>Complex Bans</h3>`,
+			`<ul><li><code>- Blaziken + Speed Boost</code> ban a combination of things in a single Pokemon (you can have a Blaziken, and you can have Speed Boost on the same team, but the Blaziken can't have Speed Boost)</li>`,
+			`<li><code>- Drizzle ++ Swift Swim</code> ban a combination of things in a team (if any Pokémon on your team have Drizzle, no Pokémon can have Swift Swim)</li></ul>`,
+			`<h2><u>Unbans</u></h2>`,
+			`<p>Using a <code>+</code> instead of a <code>-</code> Unbans that category.</p>`,
```

Comment:
```suggestion
			`<p>Using a <code>+</code> instead of a <code>-</code> unbans that category.</p>`,
```

Like, for instance, this is grammatically incorrect.